chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,786 @@
+	a sequence of unrelated numbers, i.e. a random process. 
+	\textit{Randomization} will never be used to mean the process of assigning units in treatment and control groups,
+	that will always be called \textit{random treatment assignment},
+	or a derivative thereof.} 
+is used to ensure that a sample is representative and
+that any treatment and control groups are statistically indistinguishable
+after treatment assignment.
+
+Randomization in statistical software is non-trivial
+and its mechanics are unintuitive for the human brain.
+The principles of randomization we will outline
+apply not just to random sampling and random assignment,
+but they apply to all statistical computing processes that have a random components
+such as simulations and bootstrapping.
+Furthermore, all random processes introduce statistical noise
+or uncertainty into estimates of effect sizes.
+Choosing one random sample from all the possibilities produces some probability of
 choosing a group of units that are not, in fact, representative.
-Choosing one final randomization assignment similarly produces some probability of
-creating groups that are not good counterfactuals for each other.
-Power calculation and randomization inference
+Similarly, choosing one random assignment produces some probability of
+creating groups that are not good counterfactuals.
+\textit{Power calculation} and \textit{randomization inference}
 are the main methods by which these probabilities of error are assessed.
-These analytical dimensions are particularly important in the initial phases of development research --
-typically conducted well before any actual field work occurs --
-and often have implications for feasibility, planning, and budgeting.
+These analyses are particularly important in the initial phases of development research --
+typically conducted before any actual field work occurs --
+and have implications for feasibility, planning, and budgeting.
 
-In this chapter, we first cover the necessary practices to ensure that random processes are reproducible.
-We next turn to how to implement sampling and randomized assignment,
-both for simple, uniform probability cases, and more complex designs,
-such as those that require clustering or stratification.
-We include code examples so the guidance is concrete and applicable.
-The last section discusses power calculations and randomization inference,
-and how both are important tools to critically and quantitatively assess different
-sampling and randomization designs and to make optimal choices when planning studies.
+%-----------------------------------------------------------------------------------------------
+\subsection{Randomizing sampling and treatment assignment}
+
+% sampling universe: the master dataset
+\textbf{Sampling} is the process of randomly selecting units-of-observation
+from a list of individuals to create a representative or statistically similar sub-sample.\index{sampling}
```

Comment:
```suggestion
\textbf{Sampling} is the process of randomly selecting observations
from a list of individuals to create a representative or statistically similar sub-sample.\index{sampling}
```
"Selecting unit of observation" seems like choosing between household or school level